docker/Dockerfile.rtcloud

```diff
@@ -1,24 +1,20 @@
-FROM brainiak/rtconda:1.2 as CONDA_IMAGE
-
-FROM brainiak/rttools:1.2
+FROM brainiak/rtconda:1.3 as CONDA_IMAGE
+FROM brainiak/rttools:1.3
 
 SHELL ["/bin/bash", "-c"]
 
 RUN \
-mkdir /rt-cloud && \
-chown rtuser:rtuser /rt-cloud
-
-USER rtuser
+mkdir /rt-cloud
```

Comment:
I guess if you are trying to minimize 'RUN' commands for image size reasons this could be moved and combined within a lower 'RUN' command.